docs/source/modules/benchmark_tutorial.rst

```diff
@@ -177,6 +189,47 @@ assembly:
         # etc
 
 
+
+**Adding your data to Brain-Score**:
+You will also need an :code:`__init__.py` file to go along with your submission. The purpose of this file is to register the
+benchmark inside the Brain-Score ecosystem. This involves adding both the stimuli and the data to the
+:code:`stimulus_set_registry` and :code:`data_registry` respectively. See below for an example from the data for :code:`Geirhos2021`:
+
+.. code-block:: python
+
+    # assembly
+    data_registry['Geirhos2021_colour'] = lambda: load_assembly_from_s3(
+        identifier='brendel.Geirhos2021_colour',
+        version_id="RDjCFAFt_J5mMwFBN9Ifo0OyNPKlToqf",
+        sha1="258862d82467614e45cc1e488a5ac909eb6e122d",
+        bucket="brainio-brainscore",
+        cls=BehavioralAssembly,
+        stimulus_set_loader=lambda: load_stimulus_set('Geirhos2021_colour'),
+    )
+
+    # stimulus set
+    stimulus_set_registry['Geirhos2021_colour'] = lambda: load_stimulus_set_from_s3(
+        identifier='brendel.Geirhos2021_colour',
+        bucket="brainio-brainscore",
+        csv_sha1="9c97c155fd6039a95978be89eb604c6894c5fa16",
+        zip_sha1="d166f1d3dc3d00c4f51a489e6fcf96dbbe778d2c",
+        csv_version_id="1ZaFYwHPBkDOrgdrwGHYqMfJJBCWei21",
+        zip_version_id="X62ivk_UuHgh7Sd7VwDxgnB8tWPK06gt")
+
+
+**Data Packaging Summary**:
+Part 1 of creating a benchmark involves packaging the stimuli and data, adding a :code:`test.py` file, and adding these stimuli
+and data to the :code:`data_registry`. The summary of what to submit is seen below with an example structure of an example
+submission structure:
+
+.. code-block:: python
+
+    MyBenchmark2024_stimuli_and_data/
```

Comment:
maybe we want to write `<Contents of eventual zip file submission>`? To make clear that this is not a required folder structure